update_versions.conf

```diff
@@ -24,6 +24,6 @@
 # The following file does not exist in the repo as a static file.
 # It is generated at build time
 sourcefile: product-deletion-utility.version
-tag: 0.0.0-product-deletion-utility
+tag: 1.0.0
```

Comment:
```suggestion
tag: 0.0.0
```

Reply:
Thanks @mharding-hpe for the suggestion. It worked